client/verta/tests/unit_tests/strategies.py

```diff
@@ -262,3 +260,98 @@ def mock_workspace(draw):
         )
     )
     return workspace
+
+
+@st.composite
+def mock_pipeline_definition(draw):
```

Comment:
I think we should consider not calling our composite strategies `mock_*`. It can be safely assumed that a property test strategy yields a mocked value, and I'm seeing in this PR that we're risking confusion (and shadowing) between fixture and strategy names.

Reply:
refactored to remove all usage of "mock" in hypothesis strategies